src/locales/default.json

```diff
@@ -355,12 +355,14 @@
       "information": "The minimum amount of voting power required to create a proposal"
     },
     "allowOnlyAuthors": "Allow only authors to submit a proposal",
+    "enableAliased": "Allow voting with session keys",
```

Comment:
```suggestion
    "enableAliased": "Allow actions with session keys",
```

Reply:
Yes it can stay there for now. We will rework entire settings once we have the design power to do so.